flight/Modules/Actuator/actuator.c

```diff
@@ -170,98 +206,42 @@ static void actuatorTask(void* parameters)
 	FlightStatusData flightStatus;
 
 	/* Read initial values of ActuatorSettings */
-	ActuatorSettingsData actuatorSettings;
-	actuator_settings_updated = true;
-	ActuatorSettingsGet(&actuatorSettings);
-
-	/* Read initial values of MixerSettings */
-	MixerSettingsData mixerSettings;
-	mixer_settings_updated = true;
-	MixerSettingsGet(&mixerSettings);
-
-	/* Force an initial configuration of the actuator update rates */
-	actuator_update_rate_if_changed(&actuatorSettings, true);
-
-	// Go to the neutral (failsafe) values until an ActuatorDesired update is received
-	setFailsafe(&actuatorSettings, &mixerSettings);
+	settings_updated = true;
 
 	// Main task loop
 	lastSysTime = PIOS_Thread_Systime();
-	while (1)
-	{
+	while (1) {
 		PIOS_WDG_UpdateFlag(PIOS_WDG_ACTUATOR);
 
-		// Wait until the ActuatorDesired object is updated
-		bool rc = PIOS_Queue_Receive(queue, &ev, FAILSAFE_TIMEOUT_MS);
+		bool rc = false;
```

Comment:
Not your fault, but I would be a proponent for refactoring this variable name to be something sensible. ```rc``` means as much as ```tmp1``` in this context, and it actually is harder to search for than ```tmp1``` because the string "rc" shows up in lots of unrelated spots, like "ActuatorCommand"